### ISSUE TITLE
Feedback form button accessibility

### ISSUE THREAD

**Issue by jamiepantling** [CivicTechTO/civictechto.github.io#82]
Colour contrast on buttons needs amending to make fully accessible. e.g. black on brown

<img width="185" height="118" alt="Image" src="https://github.com/user-attachments/assets/038222c6-27e3-4a81-acd7-2d8abe673085" />
